planet/scripts/cli.py

```diff
@@ -62,8 +75,12 @@ def configure_logging(verbosity):
 @click.option('-u', '--base-url', envvar='PL_API_BASE_URL',
               help='Change the base Planet API URL or ENV PL_API_BASE_URL'
                    ' - Default https://api.planet.com/')
+@click.option('-u', '--analytics-base-url', envvar='PL_ANALYTICS_API_BASE_URL',
```

Comment:
this is re-using the same short option name (`-u`) as `base-url` above, is `analytics-base-url` supposed to function as an alias?

Reply:
Oh, good catch, it shouldn't be an alias as the two may need to be controlled separately.

Reply:
Alright, I just made it `-au` so that it won't overlap.